store/extensions/GPT Reply/extension.json

```
@@ -0,0 +1,38 @@
+{
+    "title": "GPT Reply",
+    "description": "GPT Reply lets you reply to other people with Claude or ChatGPT in Origins.",
+    "authors": [
+      {
+        "name": "QDave",
+        "discord": "qdave"
+      }
+    ],
+    "version": "1.0",
+    "categories": ["Usability"],
+    
+    "source": "https://github.com/QDaves/GPTReply",
+    "readme": "https://github.com/QDaves/GPTReply/blob/main/README.md",
+    "releases": "https://github.com/QDaves/GPTReply/releases",
+    
+    "stable": true,
+    
+    "framework": {
+      "name": "GoEarth",
+      "version": "0.3.3"
+    },
+  
+    "language": "Go",
+    "commands": {
+      "default": ["cmd", "/C", "GPT Reply-win.exe", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"],
+      "windows": ["cmd", "/C", "GPT Reply-win.exe", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"]
+    },

```

You have print statements in your code, best to either remove them, wrap them in a debug check or pipe the extensions output to nul in this command.
```suggestion
    "commands": {
      "default": ["cmd", "/C", "GPT Reply-win.exe", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}", ">nul", "2>&1"],
      "windows": ["cmd", "/C", "GPT Reply-win.exe", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}", ">nul", "2>&1"]
    },

```